Header and sidebar test

SurajMohe opened the pull request.

yashpriyam committed:
Create README.md

SurajMohe committed:
complete\

Sachinrawat1215 commented on index.html:
```diff
@@ -0,0 +1,39 @@
+     <meta charset="UTF-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link
+    href="https://cdn.jsdelivr.net/npm/remixicon@4.2.0/fonts/remixicon.css"
```
You didn't need to add any external library.

Sachinrawat1215 commented on index.html:
```diff
@@ -0,0 +1,39 @@
+    rel="stylesheet"
+/>
+     <link rel="stylesheet" href="style.css">
+     <title>Document</title>
```
Add a proper title for the HTML file

Sachinrawat1215 commented on style.css:
```diff
@@ -0,0 +1,91 @@
+
+html,
+body {
+     overflow: hidden;
```
Why you need to set overflow hidden here?

Sachinrawat1215 commented on style.css:
```diff
@@ -0,0 +1,91 @@
+     align-items: center;
+     justify-content: space-between;
+     width: 100%;
+     height: 15%;
```
Don't give height in % otherwise it will break on too small and large devices.

SurajMohe committed:
complete the task and some changes here

SurajMohe commented:
I corrected and updated the code after seeing the mistakes

On Sat, 20 Apr, 2024, 11:10 pm Sachin Rawat, ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In index.html
> <https://github.com/WMB2-29012024/SURAJ_MOHE_B2/pull/30#discussion_r1573351006>
> :
>
> > @@ -0,0 +1,39 @@
> +
> +<!DOCTYPE html>
> +<html lang="en">
> +<head>
> +     <meta charset="UTF-8">
> +     <meta name="viewport" content="width=device-width, initial-scale=1.0">
> +     <link
> +    ***@***.***/fonts/remixicon.css"
>
> You didn't need to add any external library.
> ------------------------------
>
> In index.html
> <https://github.com/WMB2-29012024/SURAJ_MOHE_B2/pull/30#discussion_r1573351095>
> :
>
> > @@ -0,0 +1,39 @@
> +
> +<!DOCTYPE html>
> +<html lang="en">
> +<head>
> +     <meta charset="UTF-8">
> +     <meta name="viewport" content="width=device-width, initial-scale=1.0">
> +     <link
> +    ***@***.***/fonts/remixicon.css"
> +    rel="stylesheet"
> +/>
> +     <link rel="stylesheet" href="style.css">
> +     <title>Document</title>
>
> Add a proper title for the HTML file
> ------------------------------
>
> In style.css
> <https://github.com/WMB2-29012024/SURAJ_MOHE_B2/pull/30#discussion_r1573351568>
> :
>
> > @@ -0,0 +1,91 @@
> +* {
> +     margin: 0;
> +     padding: 0;
> +     box-sizing: border-box;
> +}
> +
> +html,
> +body {
> +     overflow: hidden;
>
> Why you need to set overflow hidden here?
> ------------------------------
>
> In style.css
> <https://github.com/WMB2-29012024/SURAJ_MOHE_B2/pull/30#discussion_r1573351942>
> :
>
> > +
> +html,
> +body {
> +     overflow: hidden;
> +     width: 100%;
> +     height: 100%;
> +}
> +
> +#nav {
> +     font-family: gilroy;
> +     padding: 0px 20px;
> +     display: flex;
> +     align-items: center;
> +     justify-content: space-between;
> +     width: 100%;
> +     height: 15%;
>
> Don't give height in % otherwise it will break on too small and large
> devices.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/WMB2-29012024/SURAJ_MOHE_B2/pull/30#pullrequestreview-2013124284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDQZ36SITQEJXR2GZWSVLYTY6KSB7AVCNFSM6AAAAABGPFHAFKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAMJTGEZDIMRYGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
